cypress/e2e/signIn.cy.js

```diff
@@ -1,34 +1,32 @@
-/// <reference types='cypress' />
-/// <reference types='../support' />
-
 import SignInPageObject from '../support/pages/signIn.pageObject';
 import HomePageObject from '../support/pages/home.pageObject';
 
-const signInPage = new SignInPageObject();
-const homePage = new HomePageObject();
+const SignInPage = new SignInPageObject();
+const HomePage = new HomePageObject();
 
 describe('Sign In page', () => {
   let user;
 
   before(() => {
     cy.task('db:clear');
+  });
+
+  beforeEach(() => {
     cy.task('generateUser').then((generateUser) => {
       user = generateUser;
     });
+    SignInPage.visit();
   });
 
   it('should provide an ability to log in with existing credentials', () => {
-    signInPage.visit();
-    cy.register(user.email, user.username, user.password);
-
-    signInPage.typeEmail(user.email);
-    signInPage.typePassword(user.password);
-    signInPage.clickSignInBtn();
-
-    homePage.assertHeaderContainUsername(user.username);
+    cy.register(user);
+    SignInPage.fillSignInForm(user);
+    HomePage.assertHeaderContainUsername(user.username);
   });
 
   it('should not provide an ability to log in with wrong credentials', () => {
-
+    SignInPage.fillSignInForm(user);
+    SignInPage.clickSignInBtn();
+    cy.contains('Invalid user credentials').should('be.visible');
   });
```

Comment:
In the 'should not provide an ability to log in with wrong credentials' test, ensure that the credentials used are incorrect. Currently, it uses the generated user credentials, which may be valid. Modify the credentials to ensure they are incorrect for this test.